ui/app/styles/components/selectable-card-container.scss

```diff
@@ -14,6 +14,10 @@
     min-width: 350px;
     margin-left: auto;
     margin-right: auto;
+
+    &.wide {
+      max-width: 40%;
+    }
```

Comment:
For this instance it increased the width enough for the button text to stay on a single line but I wonder if that would be the case for buttons with longer text? I wonder if this specific card needs to have the max width of 33% or could it just expand to fit the content?